### PR TITLE
Coachmark reposition on page resize 

### LINKED ISSUE #25415: [Bug]: Coachmark and TeachingBubble anchors don't handle page resizing
### Library

React / v8 (@fluentui/react)

### System Info

```shell
"@fluentui/react": "^8.85.0"
```

### Are you reporting Accessibility issue?

_No response_

### Reproduction

Here's an easy repro:

- v7 https://codesandbox.io/s/funny-dew-eihn2
- v8 https://codesandbox.io/s/sleepy-alex-2e25bi

### Bug Description

Actual behavior:
At initial zoom, coachmark looks like:
(browser is 'Restore down')
image

After zooming out, coachmark looks like:
(browser is maximized)
image

Please note, the issue is happening on page resizing

Expected behavior:
Callout elements remain anchored to their targets.

### Logs

_No response_

### Requested priority

High

### Products/sites affected

Excel

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes #25415

Resizing the page via page zoom will recalculate the position of the PositioningContainer, but it will not update the bounds passed into the container. This causes layout errors when zooming out.

This change will re-render the coachmark on page resize, passing in the correct bounds value, and ensuring the the new bounds value is used when calculating the new coachmark position.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0b35790d67cad33bd1d52d2418fcca6ef8b4a710'>0b35790d67cad33bd1d52d2418fcca6ef8b4a710</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-92e3b62a-42e0-401c-be50-be5ee1c7240e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-92e3b62a-42e0-401c-be50-be5ee1c7240e.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26725&commitId=df778e39c0c3daaa507128377960e6a391372506&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.default.chromium.png|159| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-mount_base.html">1694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-mount.html">1996</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TeachingBubble-mount_base.html">85515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TeachingBubble-mount.html">90488</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/BaseButton-mount_base.html">1198</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/BaseButton-mount.html">1194</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Breadcrumb-mount_base.html">2590</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Breadcrumb-mount.html">2977</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Checkbox-mount_base.html">2511</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Checkbox-mount.html">2523</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/CheckboxBase-mount_base.html">2291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/CheckboxBase-mount.html">2278</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ChoiceGroup-mount_base.html">4224</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ChoiceGroup-mount.html">4107</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ComboBox-mount_base.html">1125</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ComboBox-mount.html">1137</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/CommandBar-mount_base.html">9313</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/CommandBar-mount.html">9382</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ContextualMenu-mount_base.html">14645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ContextualMenu-mount.html">14401</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DefaultButton-mount_base.html">1394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DefaultButton-mount.html">1259</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DetailsRow-mount_base.html">3266</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DetailsRow-mount.html">3482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DetailsRowFast-mount_base.html">3481</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DetailsRowFast-mount.html">3439</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DetailsRowNoStyles-mount_base.html">3251</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DetailsRowNoStyles-mount.html">3168</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Dialog-mount_base.html">3026</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Dialog-mount.html">2976</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DocumentCardTitle-mount_base.html">575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/DocumentCardTitle-mount.html">516</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Dropdown-mount_base.html">3019</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Dropdown-mount.html">3006</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/FocusTrapZone-mount_base.html">1910</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/FocusTrapZone-mount.html">1840</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/FocusZone-mount_base.html">1953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/FocusZone-mount.html">1896</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-mount_base.html">1694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-mount.html">1996</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-virtual-rerender_base.html">1082</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-virtual-rerender.html">1006</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedList-virtual-rerender-with-unmount.html">1605</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedListV2-mount_base.html">539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedListV2-mount.html">571</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedListV2-virtual-rerender_base.html">480</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedListV2-virtual-rerender.html">542</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">552</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">516</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/IconButton-mount_base.html">1838</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/IconButton-mount.html">1848</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Label-mount_base.html">697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Label-mount.html">725</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Layer-mount_base.html">4169</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Layer-mount.html">4088</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Link-mount_base.html">835</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Link-mount.html">825</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/MenuButton-mount_base.html">1677</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/MenuButton-mount.html">1637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/MessageBar-mount_base.html">2328</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/MessageBar-mount.html">2093</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Nav-mount_base.html">3199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Nav-mount.html">3060</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/OverflowSet-mount_base.html">1352</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/OverflowSet-mount.html">1354</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Panel-mount_base.html">2530</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Panel-mount.html">2504</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Persona-mount_base.html">1250</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Persona-mount.html">1269</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Pivot-mount_base.html">1582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Pivot-mount.html">1473</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/PrimaryButton-mount_base.html">1424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/PrimaryButton-mount.html">1528</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Rating-mount_base.html">6586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Rating-mount.html">6825</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SearchBox-mount_base.html">1506</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SearchBox-mount.html">1400</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Shimmer-mount_base.html">2711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Shimmer-mount.html">2751</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Slider-mount_base.html">1946</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Slider-mount.html">2043</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SpinButton-mount_base.html">4420</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SpinButton-mount.html">4417</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Spinner-mount_base.html">796</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Spinner-mount.html">767</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SplitButton-mount_base.html">3008</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SplitButton-mount.html">3046</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Stack-mount_base.html">741</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Stack-mount.html">761</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/StackWithIntrinsicChildren-mount_base.html">2278</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/StackWithIntrinsicChildren-mount.html">2235</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/StackWithTextChildren-mount_base.html">4592</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/StackWithTextChildren-mount.html">4493</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SwatchColorPicker-mount_base.html">9936</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/SwatchColorPicker-mount.html">10005</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TagPicker-mount_base.html">2473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TagPicker-mount.html">2659</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TeachingBubble-mount_base.html">85515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TeachingBubble-mount.html">90488</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Text-mount_base.html">777</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Text-mount.html">781</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TextField-mount_base.html">1445</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/TextField-mount.html">1581</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ThemeProvider-mount_base.html">1429</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ThemeProvider-mount.html">1503</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ThemeProvider-virtual-rerender_base.html">1018</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ThemeProvider-virtual-rerender.html">976</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2087</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2049</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Toggle-mount_base.html">1107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/Toggle-mount.html">1060</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/buttonNative-mount_base.html">500</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26725/perf-test/buttonNative-mount.html">528</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by smhigley — APPROVED** [review]
I'm a bit confused about why we spread the bounds calculation across both Coachmark.base, and PositioningContainer (since PositioningContainer is a inner Coachmark control, and only used in Coachmark?).

It seems like it would be better to have bounds fully calculated in one place, and only invalidate that if e.g. the window innerHeight/Width changes, but also changing that seems potentially breaking since `PositioningContainer` is exported, so ¯\_(ツ)_/¯

**Comment by micahgodbolt** [comment]
Yeah, it was a 'fix the bug without breaking the API' type of deal

**Pull request merged**